Examples/JamitFoundationExample/JamitFoundationExample.xcodeproj/project.pbxproj

```diff
@@ -834,7 +839,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = DB7EUU9D79;
 				INFOPLIST_FILE = App/SupportingFiles/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Are we sure we want to increase the deployment target? Wasn't it even 9.0 before?

Reply:
This deployment target also only applies to the example project, not to JamitFoundation itself.